plat/kvm/Makefile.uk

```diff
@@ -134,7 +134,6 @@ LIBKVMPLAT_SRCS-y                          += $(UK_PLAT_COMMON_BASE)/memory.c|co
 LIBKVMPLAT_SRCS-y                          += $(UK_PLAT_KVM_DEF_LDS)
 ifeq ($(CONFIG_KVM_BOOT_EFI_STUB),y)
 LIBKVMPLAT_SRCS-y                          += $(LIBKVMPLAT_BASE)/efi.c|common
-LIBKVMPLAT_SRCS-y                          += $(LIBKVMPLAT_BASE)/pe_header.S|common
```

Comment:
Uuh... oops looks like I forgot to squash this commit while rebasing sorry about that.